### PR TITLE
Fix and update order validations, fix order assset mint param

### PR DESCRIPTION
**Pull request by kamilsluszniak**
Refactored order validation if's to `if (...) return` insted of nested ifology.
Added validation for owned NFT when selling.
Fixed `quoteMint: PublicKey` param passed as `string`

### CODE REVIEW

File: src/stores/GlobalStore.ts [hunk 1]
```diff
@@ -48,6 +53,7 @@ export const useGlobalStore = defineStore('globalStore', {
                     },
                 },
             ],
+            userStarAtlasNfts: [] as UserNft[],
```
Comment (skullnbonesdao) [inline src/stores/GlobalStore.ts:56]: why do we store the userNft in gloabl store?

Comment (kamilsluszniak) [reply src/stores/GlobalStore.ts:56]: Do you think it would make more sense to create dedicated store for `userStarAtlasNfts` and `userTokens`?

Comment (skullnbonesdao) [reply src/stores/GlobalStore.ts:56]: for tokens there is an object which can be used for filtering https://github.com/skullnbonesdao/trade.skullnbones.xyz/blob/master/src/typescript/constants/tokens.ts and the current selected is stored inside gloablStore() there is also a functions to change the asset by name

Comment (skullnbonesdao) [reply src/stores/GlobalStore.ts:56]: userStarAtlasNfts what do we store in there?

**Pull request merged**